gymnasium/envs/mujoco/mujoco_rendering.py

```diff
@@ -715,6 +715,11 @@ def _get_viewer(self, render_mode: Optional[str]):
         - `WindowViewer` class for "human" render mode
         - `OffScreenViewer` class for "rgb_array" or "depth_array" render mode
         """
+        if render_mode != "human":
+            assert (
+                self.width is not None and self.height is not None
+            ), "The width and height cannot be `None` when the render_mode is not Human."
```

Comment:
Could you specify what the width and height values are to help users debug
`f'The width ({width}) and height ({height}) cannot be `None`'` ...